package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Jun 12 07:16:39 UTC 2019 - Guillaume GARDET <guillaume.gardet@opensuse.org>
+
+- Update link to https://download.opensuse.org/YaST/Repos/* 
+  for non-x86
```

Comment:
please add also issue ( bsc, fate or where is more details )

Reply:
Done